app/experimenter/nimbus-ui/package.json

```diff
@@ -7,16 +7,20 @@
     "start": "SKIP_PREFLIGHT_CHECK=true react-scripts start",
     "build": "SKIP_PREFLIGHT_CHECK=true react-scripts build",
     "test": "SKIP_PREFLIGHT_CHECK=true react-scripts test --coverage",
+    "lint": "yarn lint:eslint && yarn lint:tsc && yarn lint:styles",
```

Comment:
If this is our lint now then we should update the alias in the Makefile as well:

https://github.com/mozilla/experimenter/blob/main/Makefile#L34